rust/perspective-python/perspective/legacy.py

```diff
@@ -0,0 +1,142 @@
+#  ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+#  ┃ ██████ ██████ ██████       █      █      █      █      █ █▄  ▀███ █       ┃
+#  ┃ ▄▄▄▄▄█ █▄▄▄▄▄ ▄▄▄▄▄█  ▀▀▀▀▀█▀▀▀▀▀ █ ▀▀▀▀▀█ ████████▌▐███ ███▄  ▀█ █ ▀▀▀▀▀ ┃
+#  ┃ █▀▀▀▀▀ █▀▀▀▀▀ █▀██▀▀ ▄▄▄▄▄ █ ▄▄▄▄▄█ ▄▄▄▄▄█ ████████▌▐███ █████▄   █ ▄▄▄▄▄ ┃
+#  ┃ █      ██████ █  ▀█▄       █ ██████      █      ███▌▐███ ███████▄ █       ┃
+#  ┣━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┫
+#  ┃ Copyright (c) 2017, the Perspective Authors.                              ┃
+#  ┃ ╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌╌ ┃
+#  ┃ This file is part of the Perspective library, distributed under the terms ┃
+#  ┃ of the [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0). ┃
+#  ┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
+
+# from asyncio import AbstractEventLoop
+# from typing import Optional
+from .perspective import PySyncServer, PySyncClient
+import types
+import sys
+
+# LEGACY API
+
+
+def default_loop_cb(fn, *args, **kwargs):
+    return fn(*args, **kwargs)
+
+
+def create_sync_client(cb=default_loop_cb):
+    def handle_sync_client(bytes):
+        sync_session.handle_request(bytes)
+        default_loop_cb(lambda: sync_session.poll())
+
+    def handle_new_session(bytes):
+        default_loop_cb(lambda: sync_client.handle_response(bytes))
+
+    sync_server = PySyncServer()
+    sync_session = sync_server.new_session(handle_new_session)
+    sync_client = PySyncClient(handle_sync_client)
+    return sync_client
+
+
+sync_client = create_sync_client(default_loop_cb)
+
+
+table_mod = types.ModuleType("perspective.table")
+table_mod.Table = sync_client.table
+sys.modules.setdefault("perspective.table", table_mod)
+Table = sync_client.table
+
+
+def PerspectiveManager():
+    return create_sync_client(default_loop_cb)
+
+
+# def async_client():
+#     return create_async_client(default_loop_cb)
+
+
+def set_threadpool_size(num_cpus):
```

Comment:
I think we can eliminate this, we've bumping a major version so no need for backwards compatible on this func

Reply:
The name of this file is `'legacy.py`